admin/api_clusters.go

```diff
@@ -157,6 +157,31 @@ type ClustersApi interface {
 	// Method available only for mocking purposes
 	GetSampleDatasetLoadStatusExecute(r GetSampleDatasetLoadStatusApiRequest) (*SampleDatasetStatus, *http.Response, error)
 
+	/*
+		GrantMongoDBEmployeeAccess Grant MongoDB employee cluster access for one cluster.
+
+		Grants MongoDB employee cluster access for the given duration and at the specified level for one cluster.
+
+		@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+		@param groupId Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.  **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
+		@param clusterName Human-readable label that identifies this cluster.
+		@param employeeAccessGrant Grant access level and expiration.
+		@return GrantMongoDBEmployeeAccessApiRequest
+	*/
+	GrantMongoDBEmployeeAccess(ctx context.Context, groupId string, clusterName string, employeeAccessGrant *EmployeeAccessGrant) GrantMongoDBEmployeeAccessApiRequest
```

Comment:
Confirmed that this API is user facing. Known feature in UI

Reply:
yes, confirmed, I'm working on exposing this feature in Terraform and went public to Prod yesterday